mods/default/nodes.lua

```diff
@@ -2013,10 +2013,12 @@ function default.register_chest(name, d)
 			local meta = minetest.get_meta(pos)
 			meta:set_string("formspec", nil)
 			local inv = meta:get_inventory()
-			local list = inv:get_list("main")
-			inv:set_list("main", nil)
-			inv:set_size("default:chest", 8*4)
-			inv:set_list("default:chest", list)
+			local list = inv:get_list("default:chest")
+			if list then
+				inv:set_size("main", 8*4)
+				inv:set_list("main", list)
+				inv:set_list("default:chest", nil)
+			end
```

Comment:
Hmm then it's okay with me if and only if there was no-one made builds with this version of mtg included. If someone did, I think we have to assume a number of users used it for who it isn't that easy to reset the LBMs manually, so having a separate LBM would be better I think (and, it wouldn't be very expensive either)